### DISCUSSION TITLE
C++日志如何打印查看

### DISCUSSION THREAD

**Discussion by chrwhy** [wangfenjin/simple#136] [Q&A]
请问下，C++开发如果想要调试源码要如何进行？运行过程中的日志要怎么打印，查看？

**Comment by wangfenjin** [comment]
最好还是通过单测来验证，这个库最终会放到sqlite里面执行，在里面打日志不太方便

**Reply by chrwhy** [reply]
再问一个问题，sqlite执行期间的日志有办法查看吗？

**Reply by wangfenjin** [reply]
没研究过
